toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/SceneView.kt

```diff
@@ -0,0 +1,44 @@
+package com.arcgismaps.toolkit.geocompose
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalLifecycleOwner
+import androidx.compose.ui.semantics.contentDescription
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.viewinterop.AndroidView
+import com.arcgismaps.mapping.ArcGISScene
+
+/**
+ * A compose equivalent of the view-based [SceneView].
```

Comment:
looks like this SceneView link points to itself instead of the view-based SceneView. Maybe safest to fulloy qualify it:
```suggestion
 * A compose equivalent of the view-based [com.arcgismaps.mapping.view.SceneView].
```